spec/20-http_request_header_format.md

```diff
@@ -176,6 +176,8 @@ There are two additional options that vendors MAY follow:
 - A component that makes a deferred or delayed recording decision may communicate the priority of a recording by setting `sampled` flag to `1` for a subset of requests.
 - A component may also fall back to probability sampling and set the `sampled` flag to `1` for the subset of requests.
 
+**Note**: Communicating the recording decision downstream via the `sampled` flag obviously requires adding the `traceparent` header to outgoing requests in the first place. In cases where the incoming request did not have the `traceparent` header and the tracing tool decides not to record trace data, this still entails generating a `trace-id` and `parent-id` (that is, starting a trace), even if those id values are not associated with any actual trace data.
```

Comment:
-1 to this addition. A component should be free to not send trace context downstream if there was non from upstream.

Reply:
Isn't that in contradiction to what you said [here](https://github.com/w3c/trace-context/issues/390#issuecomment-603356166)? 

How about phrasing it like this then (last sentence is new):

**Note**: Communicating the recording decision downstream via the `sampled` flag obviously requires adding the `traceparent` header to outgoing requests in the first place. In cases where the incoming request did not have the `traceparent` header and the tracing tool decides not to record trace data and to communicate that decision downstream, this still entails generating a `trace-id` and `parent-id` (that is, starting a trace), even if those id values are not associated with any actual trace data. It is also valid behavior to not start a new trace (that is, not adding the traceparent headers to outgoing requests), and accept the fact that the decision to not record trace data cannot be communicated downstream.

Reply:
I'm not against the originally proposed text as it still has some room for not doing it. It is still the actor's decision. But the text suggested in the comment is even better.

Overall the feedback was before that we need to minimize the amount of actor's behavior w.r.t. sampling in the normative section as this spec is mostly about the protocols and do not regulate how sampling will work. @yurishkuro are you partially against this section because you want to minimize the amount of details on specific sampling behavior? Maybe good alternative would be to move this notice into the non-normative section [processing model](https://github.com/w3c/trace-context/blob/master/spec/30-processing-model.md)?

Reply:
It says nothing about sampling at all though, does it? "Starting a trace" just means sending headers downstream in this context.

Reply:
> Isn't that somewhat contradictory to what you said here?

It is, indeed. New company - new perspective :-). There is a use case that even generating a new random ID is an extra unnecessary overhead for very high QPS services if the trace is not going to be sampled.

Reply:
Since Yuri now approved the changes, I guess the new phrasing resolved his concerns. Regarding the suggestion to move this into the processing model: This is very specific to the sampled flag, which is talked about here in detail, so I think it would be better to keep this particular notice in this section.